seisbench/generate/labeling.py

```diff
@@ -228,112 +241,18 @@ def label(self, X, metadata):
-            )
-        elif self.ndim == 3:
-            y[:, self.label_ids["Noise"], :] = 1 - np.nansum(y, axis=channel_dim)
-            y = self._swap_dimension_order(
-                y, current_dim="NCW", expected_dim=config["dimension_order"]
-            )
-
-        return y
-
-    def __str__(self):
-        return f"ProbabilisticLabeller (label_type={self.label_type}, dim={self.dim})"
-
-
-class TriangularLabeller(PickLabeller):
-    r"""
-    Create supervised labels from picks. The picks in example are represented
-    probabilistically with a triangular label. Class transferred from ProbabilisticLabeller
-            / \
-           /   \
-          /     \
-         /       \
-        /         \
-    ___/           \___
-       ----- | -----
-        2*half_width
-
-    All picks with NaN sample are treated as not present.
-
-    :param half_width: The half width of the triangular distribution in samples
-    :type half_width: int, optional
-    """
-
-    def __init__(self, half_width=10, **kwargs):
-        self.label_method = "probabilistic"
-        self.half_width = half_width
-        kwargs["dim"] = kwargs.get("dim", 1)
-        super().__init__(label_type="multi_class", **kwargs)
-
-    def label(self, X, metadata):
-        if not self.label_columns:
-            label_columns = self._auto_identify_picklabels(metadata)
-            (
-                self.label_columns,
-                self.labels,
-                self.label_ids,
-            ) = self._colums_to_dict_and_labels(label_columns)
-
-        sample_dim, channel_dim, width_dim = self._get_dimension_order_from_config(
-            config, self.ndim
-        )
-
-        if self.ndim == 2:
-            y = np.zeros(shape=(len(self.labels), X.shape[width_dim]))
-        elif self.ndim == 3:
-            y = np.zeros(
-                shape=(
-                    X.shape[sample_dim],
-                    len(self.labels),
-                    X.shape[width_dim],
-                )
-            )
-
-        # Construct pick labels
-        for label_column, label in self.label_columns.items():
-            i = self.label_ids[label]
-
-            if label_column not in metadata:
-                # Unknown pick
-                continue
-
-            if isinstance(metadata[label_column], (int, np.integer, float)):
-                # Handle single window case
-                onset = metadata[label_column]
-                label_val = triangular_pick(
-                    onset=onset, length=X.shape[width_dim], half_width=self.half_width
-                )
+                elif self.form == "triangular":
+                    label_val = triangular_pick(
+                        onset=onset, length=X.shape[width_dim], sigma=self.sigma
+                    )
+                else:
+                    raise NotImplementedError(
+                        "Laber of form %s not implemented." % self.form
```

Comment:
For consistency with the remaining code, could you switch this to `f"Labler of form {self.form} is not implemented."`. There is also a small typo in the first word.